spec/features/posts/visit_new_post_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+feature "visit posts new", %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.